### PR TITLE
Create game

### PR DESCRIPTION
**Pull request by tom7689**
can I make a pull request for 2 issues? (in this case server #47 and #79 )

### CODE REVIEW

File: src/main/java/ch/uzh/ifi/hase/soprafs23/controller/GameController.java [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+  GameController(GameService gameService) {
+    this.gameService = gameService;
+  }
+
+
+    @PostMapping("/games/{accessCode}")
+    @ResponseStatus(HttpStatus.CREATED)
+    @ResponseBody
+    public GameGetDTO createGame(@PathVariable int accessCode) {
```
Comment (felixmerz00) [inline src/main/java/ch/uzh/ifi/hase/soprafs23/controller/GameController.java:35]: We changed the REST specification for the POST request of the game after M2. As far as I remember we decided to pass a Lobby object (containing access code, two teams of Users, ...) to the POST request.

File: src/main/java/ch/uzh/ifi/hase/soprafs23/controller/GameController.java [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+        // create lobby
```
Comment (felixmerz00) [inline src/main/java/ch/uzh/ifi/hase/soprafs23/controller/GameController.java:36]: Little copy paste error could be removed.

**Review by felixmerz00 — COMMENTED** [review]
Looks good. But I think some code doesn't correspond to our latest plan. I added comments inside the code with potential issues.

File: src/main/java/ch/uzh/ifi/hase/soprafs23/controller/TeamController.java [hunk 3]
```diff
@@ -27,6 +29,15 @@ public class TeamController {
       this.teamService = teamService;
 
   }
+  @PostMapping("/teams")
+  @ResponseStatus(HttpStatus.CREATED)
+  @ResponseBody
+  public TeamGetDTO createTeam() {
```
[the remark this answers is not in the document]
Comment (tom7689) [reply src/main/java/ch/uzh/ifi/hase/soprafs23/controller/TeamController.java:35]: was for testing the teams with postman

**Review by felixmerz00 — APPROVED** [review]
The build does not work because one test fails. But the code looks great.

**Pull request merged**